docs/source/algorithms/PeakMatching-v1.rst

```diff
@@ -0,0 +1,207 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+This algorithm takes a table of peak centres and standard deviation and matches it to a database
+to find probable energy transitions for peaks.
+
+
+Input Table and datafile requirement
+####################################
+
+- input table must have atleast 2 tables in which one column is the peak centre and the other is the standard deviation,the names of the columns must be given if they differ from the default values: centre and sigma respectively.
```

Comment:
This sentence does not make sense

Reply:
I meant to say "input table must have atleast 2 columns in which one column is the peak centre and the other is the standard deviation,the names of the columns must be given if they differ from the default values: centre and sigma respectively."